### DISCUSSION TITLE
Next release.

### DISCUSSION THREAD

**Discussion by ChrysaChm** [textbrowser/biblioteq#99] [Q&A] [answered]
Hello,

I am interested in a resolved issue that is closed recently. Do you know when the next release will be planned? Are there any nightly or snapshot builds available that I can access? Thank you very much in advance.

**Comment by textbrowser** [comment]
10th of April. No, do not provide nightly builds or snapshots. :)

**Comment by textbrowser** [comment]
Hi again. It's much work producing three Linux bundles, one Windows bundle, and a Mac package every night or even weekly. Even if automated. I cannot have machines idling away and consuming electricity for nightly compilations. Especially for PowerPC.

**Comment by textbrowser** [comment]
3 files uploaded. 2 remaining.

**Comment by textbrowser** [comment]
Windows file remains.

**Comment by textbrowser** [comment]
Done! Yaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaay. Please do not find errors!

**Comment by ChrysaChm** [comment]
Hi @textbrowser ,

I have downloaded and used the new version just fine. No issues so far.

Thank you very much! Keep up the good work!

**Comment by ChrysaChm** [comment]
My friend, unfortunately there is a big problem. I installed the new version. But when I connect to the old sql base, it shows me very few books and not all. And when i try to open one of those books it display an error message. (see photo)
![Screenshot_2](https://user-images.githubusercontent.com/81994278/114850848-dca9fa00-9de9-11eb-8e8b-0eba28e22749.jpg)

**Comment by textbrowser** [comment]
Did you update your schema? Make a copy of your database before doing so. How do you update it? Explore.

**Comment by textbrowser** [comment]
Notice the release notes.

https://github.com/textbrowser/biblioteq/releases/tag/2021.04%2F10

**Comment by ChrysaChm** [comment]
My friend I'm trying to build a new database for the new version in order to check it. As you can see in the photos, even if I change to "other options" option with "numeric" when I enter the value 8 the program does not sort arithmetic. If I enter the value 08, it works. 
![with 0](https://user-images.githubusercontent.com/81994278/115138685-474f7580-a036-11eb-8282-5a4a0eec1bef.jpg)
![without 0](https://user-images.githubusercontent.com/81994278/115138687-47e80c00-a036-11eb-8353-86df3bbdf681.jpg)

**Comment by textbrowser** [comment]
Alright. Here's a screenshot of a test I just performed. Did you refresh the table after changing the sort algorithm? Did you save the options?

![Screenshot from 2021-04-18 08-30-17](https://user-images.githubusercontent.com/10701156/115145630-82df4500-a020-11eb-9b07-1127b415b8a6.png)

**Comment by textbrowser** [comment]
Aha, you're viewing the All category. Yes? Remember, the All category (or any other non-Books category) may contain non-book items. BQ can only apply the sort algorithm when it's in the Books category state because any other category could have a mixture of item types. So, change your category to Books. I will amend the Other Options to detail this requirement.

**Comment by textbrowser** [comment]
And you do not need to prepend values with zeros.

**Comment by textbrowser** [comment, marked as answer]
OK, added new live documentation for the next release.
